applications/MeshMovingApplication/custom_utilities/fixed_mesh_ale_utilities.cpp

```diff
@@ -192,13 +194,24 @@ namespace Kratos
 
         // Search the origin model part nodes in the virtual mesh elements and
         // interpolate the values in the virtual element to the origin model part node
-        block_for_each( rOriginModelPart.Nodes(),
-            [&]( Node<3>& rNode )
+
+        unsigned int max_results=10000;
```

Comment:
note that i am defining a larger number of results than it was used before. with 1000 it was crashing (which in turn suggests that we should tune the cell size for performance reasons)